v4/export/sql.go

```diff
@@ -204,7 +210,13 @@ func SelectAllFromTable(conf *Config, db *sql.Conn, database, table string) (Tab
 		return nil, err
 	}
 
-	query := buildSelectQuery(database, table, selectedField, buildWhereCondition(conf, ""), orderByClause)
+	var query string
+	// If all columns are generated
+	if selectedField == "" {
+		query = fmt.Sprintf("SELECT '' FROM `%s`.`%s`", database, table)
+	} else {
+		query = buildSelectQuery(database, table, selectedField, buildWhereCondition(conf, ""), orderByClause)
+	}
```

Comment:
```suggestion
	queryField := selectedField
	// If all columns are generated
	if queryField == "" {
		queryField = "''"
	}
	query := buildSelectQuery(database, table, queryField, buildWhereCondition(conf, ""), orderByClause)
```

we still need to use `buildSelectQuery` so that database and table names containing `` ` `` can still be properly escaped.